Gaijinizer patch/Map119.txt

```diff
@@ -233,28 +239,31 @@ It's really strange. How about you?#MTLed
 人間だった頃の記憶があれば、話のネタの一つにでも
  なったもんだろうけど……
 > CONTEXT: Map119/events/5/pages/1/41/Dialogue/enemy10
-However, even if you talk, what did you talk
-about ......... If you have a memory of when you
-were a human, even if it is one of the stories
-I wonder if it is...but ...#MTLed
+But what is there to talk about, anyway...?
+
+If I had any memories from when I was human,
+maybe we could work with that...#Ron
 > END STRING
 
 > BEGIN STRING
 ！？
 
  『人間だった頃』って…！？
 > CONTEXT: Map119/events/5/pages/1/47/Dialogue/kazuya
-! ? "When I was a human being" ...! ?#MTLed
+!?
+
+What do you mean by that...?!#Ron
 > END STRING
 
 > BEGIN STRING
 あたいはもともと、機織りの女だったのさ
 
 あるクソ女神のせいで、こんな悪魔に転生しちまったがね。
 > CONTEXT: Map119/events/5/pages/1/51/Dialogue/enemy10
-I was originally a woman who weaved
-Because of a certain fucking goddess, I was
-incarnated to such a demon.#MTLed
+I used to be a weaver-woman...
+
+But because of a certain bitch goddess, I was
+reincarnated in this form.#Ron
 > END STRING
```

Comment:
As we mentioned before, it's likely referencing this: https://en.wikipedia.org/wiki/Arachne

I'll be suggesting a few changes to align the two together.